netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java

```diff
@@ -128,29 +133,52 @@ public AsciiString scheme() {
     };
   }
 
+  public static ProtocolNegotiator serverTls(final SslContext sslContext) {
+    return serverTls(sslContext, null);
+  }
+
   static final class ServerTlsHandler extends ChannelInboundHandlerAdapter {
     private final ChannelHandler next;
     private final SslContext sslContext;
+    private ObjectPool<? extends Executor> executorPool;
+    private Executor executor;
 
     private ProtocolNegotiationEvent pne = ProtocolNegotiationEvent.DEFAULT;
 
-    ServerTlsHandler(ChannelHandler next, SslContext sslContext) {
+    ServerTlsHandler(ChannelHandler next, SslContext sslContext,
+        ObjectPool<? extends Executor> executorPool) {
       this.sslContext = checkNotNull(sslContext, "sslContext");
       this.next = checkNotNull(next, "next");
+      this.executorPool = executorPool;
+    }
+
+    ServerTlsHandler(ChannelHandler next, SslContext sslContext) {
+      this(next, sslContext, null);
     }
 
     @Override
     public void handlerAdded(ChannelHandlerContext ctx) throws Exception {
       super.handlerAdded(ctx);
       SSLEngine sslEngine = sslContext.newEngine(ctx.alloc());
-      ctx.pipeline().addBefore(ctx.name(), null, new SslHandler(sslEngine, false));
+      if (this.executorPool != null
+          && SystemPropertyUtil.getBoolean("io.netty.handler.ssl.openssl.useTasks", false)
+          == true) {
```

Comment:
why compare with `true`? Shouldn't `&& SystemPropertyUtil.getBoolean("io.netty.handler.ssl.openssl.useTasks", false))` be enough?